src/main/java/uk/gov/digital/ho/hocs/audit/export/infoclient/ExportViewConstants.java

```diff
@@ -2,11 +2,23 @@
 
 public interface ExportViewConstants {
 
+        enum DATE_SECTION {DAY, MONTH, YEAR};
```

Comment:
Medium: enum DATE_SECTION would be better using a constructor passing regex - binding DATE_SECTION and regex use together. Would also mean you don't need a case statement in the DateAdapter.java. This would all be mitigated using regex groups.

Reply:
This enum has been removed.